plugins/modules/files/iso_customize.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_customize
+short_description: Add/remove/change files in ISO file
+description:
+  - This module is used to add/remove/change files in ISO file.
+  - The module will always execute all I(delete_files) actions first, then all I(add_files) actions.
+  - We can change an existing file inside the ISO image by combining options I(delete_files)
+  - and I(add_files).
```

Comment:
There is no reason why you should start a new paragraph in the middle of a sentence.
```suggestion
    and I(add_files).
```

Reply:
Fix it. Thanks